content/mappings/members.yaml

```diff
@@ -0,0 +1,19 @@
+- id: coolguy1990
+  twitter: 'KunalMangaraj'
+  name: 'Kunal Mangaraj'
+  status: '🍪 Happily coding away while munching on cookies'
+  image:
+    small: https://avatars2.githubusercontent.com/u/1255397?s=100
+    medium: https://avatars2.githubusercontent.com/u/1255397?s=300
+    large: https://avatars2.githubusercontent.com/u/1255397
+  website: 'https://github.com/coolguy1990'
+
+- id: john-e
+  twitter: 'engitprabhat'
+  name: 'Engit Prabhat'
+  status: '🎩 Code magician'
+  image:
+    small: https://avatars2.githubusercontent.com/u/5802221?s=100
```

Comment:
Later on, we can change this to a single image link here and leverage sharp to do the size change on build.